packages/backend/src/open_payments/payment/incoming/routes.ts

```diff
@@ -63,9 +64,15 @@ async function getIncomingPayment(
   ctx: ReadContext
 ): Promise<void> {
   let incomingPayment: IncomingPayment | undefined
+  const clientId = ctx.grant
+    ? ctx.grant?.access[0].actions.includes(AccessAction.ReadAll)
```

Comment:
:thinking: Should the OpenAPI spec disallow actions containing both `Read` and `ReadAll` (and `List` and `ListAll`)?

Reply:
I think it makes sense to have one or the other since the `All` one includes the other.

Reply:
https://github.com/interledger/open-payments/issues/178